libs/vault/src/cipher-view/custom-fields/custom-fields-v2.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="tw-mb-6">
+  <h5 class="tw-pl-2 tw-font-bold tw-text-headers tw-mb-2">{{ "customFields" | i18n }}</h5>
+  <bit-card>
+    <div
+      class="tw-mb-4 tw-border-secondary-300 tw-bg-background"
+      *ngFor="let field of fields; let last = last"
+      [class.tw-border-0]="!last"
+      [class.tw-border-b]="!last"
+      [class.tw-border-solid]="!last"
+      [class.tw-pb-2]="!last"
+      [class.tw-mb-4]="!last"
```

Comment:
🎨 We should be able to simplify this a bit with this syntax:
```suggestion
      [class]="{'tw-boder-0 tw-border-b tw-border-solid tw-pb-2 tw-mb-4': !last}"
```